app/containers/App.js

```diff
@@ -1,23 +1,23 @@
-import React, { Component, PropTypes } from 'react'
+import React from 'react'
+const DevTools = require('./DevTools')
 
-export default class App extends Component {
-  static propTypes = {
-    children: PropTypes.element.isRequired,
-  }
+const App = (props) => {
+  return (
+    <div>
+      { props.children }
+      {
+        (() => {
```

Comment:
This looks weird. Maybe assign the function in the top of the function and point to it here?